### ISSUE TITLE
Sharemap feature for wms

### ISSUE THREAD

**Issue by jokd** [origo-map/origo#1898]
**Is your feature request related to a problem? Please describe.**
It is useful if the selected feature is added to the sharemap url also for wms layers.

**Comment by Grammostola** [comment]
Cheers for creating the issue we discussed last week. I was going to if we (Eskilstuna) were going to implement it as it comes with requirements for the back end. May have an update on this tomorrow.

**Comment by jokd** [comment]
@Grammostola Any progress on this?

**Comment by Grammostola** [comment]
@jokd Potentially, there might be some news toward the end of the week. 

As for form I'm still (loosely) thinking the shared URL shall contain the layer and feature id as well as a position and the opened shared URL shall issue a getfeatureinfo/appjson for geom for selection and an infowindow/overlay depending on what the map is configured for. 

I resurrected the above mentioned issue since we (Eskilstuna municipality) expect an infowindow/overlay with text/html-content as well as a selection, which is what happens now when one infoclicks a WMS feature in our map, which is to say to getfeatureinfo requests. 

As such it seems appropriate to introduce a text/html infoFormat for WMS layers.
(One must always be vigilant as far as relatively many requests are concerned, as such a text/html infoFormat should optimally be combinable with application/json (but not automatically combined).

**Comment by johnnyblasta** [comment]
@Grammostola did this idea of sharemap feature for wms get stuck on something?

**Comment by Grammostola** [comment]
@johnnyblasta Yes, its planned implementation did at the end of April when certain conditions changed here.  We will not be working on it in the immediate future.
